include/aws/common/assert.h

```diff
@@ -18,6 +18,24 @@ void aws_fatal_assert(const char *cond_str, const char *file, int line) AWS_ATTR
 
 AWS_EXTERN_C_END
 
+#if defined(CBMC)
+#    define AWS_PANIC_OOM(mem, msg)                                                                                    \
+        do {                                                                                                           \
+            if (!(mem)) {                                                                                              \
+                fprintf(stderr, "%s: %s, line %d", msg, __FILE__, __LINE__);                                           \
+                exit(-1);                                                                                              \
```

Comment:
Why `exit(-1)` and not return `AWS_OP_FAILURE` or `NULL` (depending on the caller) and let the caller handle an appropriate response in case of failure?

Reply:
not sure I understand. The entire point is to never let NULL escape an allocation call.

Reply:
there is no appropriate response to an allocation failure. Doesn't exist